js_modules/dagster-ui/packages/ui-core/src/assets/auto-materialization/AutomaterializationEvaluationHistoryTable.tsx

```diff
@@ -5,108 +5,39 @@ import {
+  loading: boolean;
+  ticks: AssetDaemonTickFragment[];
+  statuses: Set<InstigationTickStatus>;
+  setStatuses: (statuses: Set<InstigationTickStatus>) => void;
+  setSelectedTick: (tick: AssetDaemonTickFragment | null) => void;
+  setTableView: (view: 'evaluations' | 'runs') => void;
+  paginationProps: CursorPaginationProps;
+}
 
 export const AutomaterializationEvaluationHistoryTable = ({
+  loading,
+  ticks,
+  statuses,
+  setStatuses,
   setSelectedTick,
   setTableView,
-  setTimerange,
-  setParentStatuses,
-}: {
-  setSelectedTick: (tick: AssetDaemonTickFragment | null) => void;
-  setTableView: (view: 'evaluations' | 'runs') => void;
-  setTimerange: (range?: [number, number]) => void;
-  setParentStatuses: (statuses?: InstigationTickStatus[]) => void;
-}) => {
-  const [statuses, setStatuses] = useQueryPersistedState<Set<InstigationTickStatus>>({
-    queryKey: 'statuses',
-    decode: React.useCallback(({statuses}: {statuses?: string}) => {
-      return new Set<InstigationTickStatus>(
-        statuses
-          ? JSON.parse(statuses)
-          : [
-              InstigationTickStatus.STARTED,
-              InstigationTickStatus.SUCCESS,
-              InstigationTickStatus.FAILURE,
-              InstigationTickStatus.SKIPPED,
-            ],
-      );
-    }, []),
-    encode: React.useCallback((raw: Set<InstigationTickStatus>) => {
-      return {statuses: JSON.stringify(Array.from(raw))};
-    }, []),
-  });
-
-  const {queryResult, paginationProps} = useCursorPaginatedQuery<
-    AssetDaemonTicksQuery,
-    AssetDaemonTicksQueryVariables
-  >({
-    query: ASSET_DAEMON_TICKS_QUERY,
-    variables: {
-      statuses: React.useMemo(() => Array.from(statuses), [statuses]),
-    },
-    nextCursorForResult: (data) => {
-      const ticks = data.autoMaterializeTicks;
-      if (!ticks.length) {
-        return undefined;
-      }
-      return ticks[PAGE_SIZE - 1]?.id;
-    },
-    getResultArray: (data) => {
-      if (!data?.autoMaterializeTicks) {
-        return [];
-      }
-      return data.autoMaterializeTicks;
-    },
-    pageSize: PAGE_SIZE,
-  });
-  // Only refresh if we're on the first page
-  useQueryRefreshAtInterval(queryResult, 10000, !paginationProps.hasPrevCursor);
-
-  React.useEffect(() => {
-    if (paginationProps.hasPrevCursor) {
-      const ticks = queryResult.data?.autoMaterializeTicks;
-      if (ticks && ticks.length) {
-        const start = ticks[ticks.length - 1]?.timestamp;
-        const end = ticks[0]?.endTimestamp;
-        if (start && end) {
-          setTimerange([start, end]);
-        }
-      }
-    } else {
-      setTimerange(undefined);
-    }
-  }, [paginationProps.hasPrevCursor, queryResult.data?.autoMaterializeTicks, setTimerange]);
-
-  React.useEffect(() => {
-    if (paginationProps.hasPrevCursor) {
-      setParentStatuses(Array.from(statuses));
-    } else {
-      setParentStatuses(undefined);
-    }
-  }, [paginationProps.hasPrevCursor, setParentStatuses, statuses]);
```

Comment:
This chunk of code has been moved to the newly crated `InstanceAutomaterializationEvaluationHistoryTable.tsx`, which renders this component.